src/app/FeedbackForm.js

```diff
@@ -1,30 +1,105 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const StyledForm = styled.form`
+const Grid = styled.div`
   display: grid;
-  grid-template-rows: 150px 60px;
+  justify-items: stretch;
+  align-items: stretch;
   padding: 10px;
+  overflow: scroll;
+`;
+
+const StyledForm = styled.form`
+  display: grid;
+  grid-gap: 10px;
 `;
 
 const StyledQuestionContainer = styled.div`
   display: grid;
-  grid-template-rows: 90px 40px;
-  padding: 0px 5px;
-  margin-bottom: 20px;
+  grid-template-rows: auto auto auto;
```

Comment:
you could also write `grid-template-rows: repeat(3, auto);` but maybe you can also just write `grid-template-rows: auto;`